catroid/src/org/catrobat/catroid/content/XmlHeader.java

```diff
@@ -67,94 +67,96 @@
 	@SuppressWarnings("unused")
 	private String userHandle = "";
 
+	@SuppressWarnings("PMD.UnnecessaryConstructor")
 	public XmlHeader() {
+		super();
 	}
 
-	String getProgramName() {
+	public String getProgramName() {
```

Comment:
Hmm, I can't find the pmd rule in the docu at the moment, but
- Adding private if the access modifier is missing is a good idea :wink: 
- Adding public if the access modifier is missing isn't a good idea :wink: because there is also package-private.